test/e2e/space_test.go

```diff
@@ -128,29 +125,106 @@ func TestSpace(t *testing.T) {
-				// along with its associated namespace
-
+			t.Run("unable to delete space", func(t *testing.T) {
 				// when
-				err = hostAwait.Client.Delete(context.TODO(), space)
+				err = hostAwait.Client.Delete(context.TODO(), s)
 
-				// then
+				// then it should fail while the member cluster is unknown (ie, unreachable)
 				require.NoError(t, err)
-				_, err = hostAwait.WaitForSpace(space.Name, wait.UntilSpaceHasConditions(TerminatingFailed("Cannot delete NSTemplateSet: unknown target member cluster: 'unknown'")))
+				s, err = hostAwait.WaitForSpace(s.Name, wait.UntilSpaceHasConditions(TerminatingFailed("cannot delete NSTemplateSet: unknown target member cluster: 'unknown'")))
 				require.NoError(t, err)
+
+				t.Run("update target cluster to unblock deletion", func(t *testing.T) {
+					// given
+					s.Spec.TargetCluster = memberAwait.ClusterName
+					// when
+					err = hostAwait.Client.Update(context.TODO(), s)
+
+					// then it should fail while the member cluster is unknown (ie, unreachable)
+					require.NoError(t, err)
+
+					t.Run("space should be finally deleted", func(t *testing.T) {
+						// when
+						err = hostAwait.WaitUntilSpaceDeleted(s.Name)
+						// then
+						require.NoError(t, err)
+					})
+				})
 			})
 		})
 	})
 }
 
+func TestUpdateSpace(t *testing.T) {
+
+	// given
+
+	// make sure everything is ready before running the actual tests
+	awaitilities := WaitForDeployments(t)
+	hostAwait := awaitilities.Host()
+	memberAwait := awaitilities.Member1()
+
+	rand.Seed(time.Now().UnixNano())
+
+	space := &toolchainv1alpha1.Space{
+		ObjectMeta: v1.ObjectMeta{
+			Namespace: hostAwait.Namespace,
+			Name:      "oddity-" + strconv.Itoa(rand.Int()),
+		},
+		Spec: toolchainv1alpha1.SpaceSpec{
+			TargetCluster: memberAwait.ClusterName,
+			TierName:      "base",
+		},
+	}
+
+	// when
+	err := hostAwait.Client.Create(context.TODO(), space)
+
+	// then
+	require.NoError(t, err)
+	// wait until NSTemplateSet has been created and Space is in `Ready` status
+	space, err = hostAwait.WaitForSpace(space.Name,
+		wait.UntilSpaceHasConditions(Provisioned()),
+		wait.UntilSpaceHasStatusTargetCluster(memberAwait.ClusterName),
+	)
+	require.NoError(t, err)
+	// wait until NSTemplateSet has been created and Space is in `Ready` status
+	nsTmplSet, err := memberAwait.WaitForNSTmplSet(space.Name, wait.UntilNSTemplateSetHasConditions(Provisioned()))
+	require.NoError(t, err)
+	tierChecks, err := tiers.NewChecks("base")
+	require.NoError(t, err)
+	tiers.VerifyGivenNsTemplateSet(t, memberAwait, nsTmplSet, tierChecks, tierChecks, tierChecks.GetExpectedTemplateRefs(hostAwait))
+
+	t.Run("update tier", func(t *testing.T) {
+		// given
+		ctr := NewChangeTierRequest(hostAwait.Namespace, space.Name, "advanced")
+
+		// when
+		err = hostAwait.Client.Create(context.TODO(), ctr)
+
+		// then
+		require.NoError(t, err)
+		err = hostAwait.WaitUntilChangeTierRequestDeleted(ctr.Name)
```

Comment:
We don't have to wait for the deletion of the ChangeTierRequest here - it can take a few seconds. I would move this at the very end of this test and here do this:
```suggestion
		_, err := hostAwait.WaitForChangeTierRequest(ctr.Name, toBeComplete)
		require.NoError(t, err)
```